src/app/components/administration/admin-settings/admin-settings-user-terms/admin-settings-user-terms.component.ts

```diff
@@ -10,14 +12,31 @@ import {userTermsDataStore} from "../../../../shared/store/user-terms-data-store
 export class AdminSettingsUserTermsComponent {
   markdownContent: string = '';
   markdownData: any = userTermsDataStore;
+  saveBtnDisabled: boolean = false
+
+  constructor(
+    private userTermsService: UserTermsService,
+    private matSnackBar: MatSnackBar
+  ){}
 
   ngOnInit(): void {
-    if (this.markdownData !== null) {
-      this.markdownContent = this.markdownData[0].content;
-    }
+    this.loadUserTerms()
+  }
+
+  loadUserTerms() {
+    this.userTermsService.getAllUserTerms().subscribe(
+      (data: any) => {
+        this.markdownData = data;
+        if (this.markdownData !== null && this.markdownData.length > 0) {
+          this.markdownContent = this.markdownData[0].content;
+        }
+        return this.markdownData
+      }
+    )
   }
 
-  log() {
-    console.log(this.markdownContent)
+  loadDate() {
```

Comment:
create a method to load current date from system and make it simple date format.